uber/models.py

```diff
@@ -856,6 +862,7 @@ def model_mixin(cls, model):
 
 
 class Group(MagModel, TakesPaymentMixin):
+    public_id    = Column(UUID, default=lambda: str(uuid4()))
```

Comment:
Though I maintain that we'll never accidentally see a collision here, we should add a unique constraint to these public uuid fields, to protect against bugs where we might accidentally assign one record's public id to another record.  Unless that's explicitly a feature we want, which I don't believe it is.